### PR TITLE
ENU/FLU: convert hexapod and soccer demo samples

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Worlds:
* [x] (085) projects/samples/demos/worlds/hexapod.wbt
* [x] (086) projects/samples/demos/worlds/soccer.wbt

### CODE REVIEW

File: projects/samples/demos/worlds/hexapod.wbt [hunk 1]
```diff
@@ -231,51 +230,51 @@ DEF HEXAPOD Robot {
...
           minPosition -0.7
           maxPosition 0.7
         }
       ]
       endPoint Solid {
-        translation 0.2 0 -0.05
+        translation 0.2 -0.05 0
         children [
           USE HIP_SHAPE
           DEF LEFT_FEMUR_TRANS Transform {
```
Comment (BenjaminDeleze) [inline projects/samples/demos/worlds/hexapod.wbt:250]: Shouldn't it be RIGHT_FEMUR_TRANS ?
Same remark for line 256.

If it is the case, the USEs have to be updated accordingly

**Review by BenjaminDeleze — CHANGES_REQUESTED** [review]
Seems good. I have only one minor remark

File: projects/samples/demos/worlds/hexapod.wbt (hunk 1 above)
Comment (stefaniapedrazzi) [reply projects/samples/demos/worlds/hexapod.wbt]: You are right.
I fixed some of the DEF names but not all of them

**Review by BenjaminDeleze — APPROVED** [review]
Thank you!

**Pull request merged**